src/lib/Components/Inbox/Conversations/Preview/InvoicePreview.tsx

```diff
@@ -63,15 +65,25 @@ const CostLabel = styled(PaymentRequest)`
 `
 
 interface Props extends RelayProps {
-  onSelected?: () => void
+  onSelected: () => void
+  conversationId: string
+  relay?: RelayRefetchProp
 }
 
 interface InvoiceStateButtonProps {
-  state: string | null
+  invoiceState: string | null
```

Comment:
Can this really be `null`? If so, what does that mean, is it the same as `"UNPAID"` ?